### PR TITLE
730 - tree is empty after deleting a page with sub-pages

### PR DESCRIPTION
**Pull request by flaviaF**
https://github.com/backbee/backbee-standard/issues/730

### CODE REVIEW

File: NestedNode/Repository/PageRepository.php [hunk 1]
```diff
@@ -807,6 +807,8 @@ public function toTrash(Page $page)
                     ->getRepository('BackBee\NestedNode\Section')
                     ->createQueryBuilder('n')
                     ->select('n._uid')
+                    ->where('n._uid != :curentPage')
+                    ->setParameter('curentPage', $page->getUid())
                     ->andIsDescendantOf($page->getSection());
```
Comment (crouillon) [inline NestedNode/Repository/PageRepository.php]: hum, we're looking for Section here and you exclude a Page, could you rather try `->andIsDescendantOf($page->getSection(), true)` that should exclude the current section, what we need if I understand

File: NestedNode/Tests/Repository/PageRepositoryTest.php [hunk 2]
```diff
@@ -748,13 +748,9 @@ public function testToTrash()
         $page2 = $this->repository->find('page2');
         $page3 = $this->repository->find('page3');
 
-        $this->assertEquals($page3, $this->repository->toTrash($page3));
-        $this->assertEquals(Page::STATE_DELETED, $page3->getState());
```
Comment (crouillon) [inline NestedNode/Tests/Repository/PageRepositoryTest.php]: You can't do that, if $page3 is put in the trash then its state is deleted

File: NestedNode/Tests/Repository/PageRepositoryTest.php [hunk 3]
```diff
@@ -748,13 +748,9 @@ public function testToTrash()
...
-
         $this->assertEquals($section1, $this->repository->toTrash($section1));
         self::$em->refresh($page1);
         self::$em->refresh($page2);
-        $this->assertEquals(Page::STATE_DELETED, $section1->getState());
```
Comment (crouillon) [inline NestedNode/Tests/Repository/PageRepositoryTest.php]: Same thing with $section1, so I think that we musn't exclude the current section in toTrash method()

**Comment by crouillon** [comment]
Thx @flaviaF

**Comment by gobjila** [comment]
Thanks @flaviaF

**Pull request merged**